src/app/home-page/state/recent-events.service.ts

```diff
@@ -1,24 +1,44 @@
 import { Injectable } from '@angular/core';
 import { ID } from '@datorama/akita';
-import { Event, EventsService } from 'app/shared/openapi';
+import { Event, EventsService, User } from 'app/shared/openapi';
 import { finalize, tap } from 'rxjs/operators';
 import { RecentEventsStore } from './recent-events.store';
 
 @Injectable({ providedIn: 'root' })
 export class RecentEventsService {
   constructor(private recentEventsStore: RecentEventsStore, private eventsService: EventsService) {}
 
-  get() {
+  get(user?: User) {
     this.recentEventsStore.setLoading(true);
-    this.eventsService
-      .getEvents('ALL_STARRED')
-      .pipe(
-        finalize(() => this.recentEventsStore.setLoading(false)),
-        tap((allStarredEvents) => {
-          this.recentEventsStore.set(allStarredEvents);
-        })
-      )
-      .subscribe();
+    // const events = user ? this.eventsService.getUserEvents(user.id, 'PROFILE') : this.eventsService.getEvents('PROFILE');
```

Comment:
probably doesn't make sense to check in commented code 
(unless there is a very specific situation in which is could be uncommented, in which case there should be a `//TODO comment` or similar)